### PR TITLE
Add examples for JS

### PR DESCRIPTION
**Pull request by fs-eire**

### CODE REVIEW

**Review by gwang-msft — COMMENTED** [review]
Can you also modify the /Readme.md to add link to the js folder and a brief description of the examples?

**Review by gwang-msft — COMMENTED** [review]
Please include header in all source files
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
```

File: js/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# ONNX Runtime JavaScript examples
+
+## Summary
+
+This folder contains several JavaScript examples. Most of the examples, unless remarked explicitly, are available in all NPM packages as described below:
+
+- [onnxruntime-node](https://github.com/microsoft/onnxruntime/tree/master/js/node): Node.js binding for ONNXRuntime. Can be used in Node.js applications and Node.js compatible environment (eg. Electron.js).
+- [onnxruntime-web](https://github.com/microsoft/onnxruntime/tree/master/js/web): ONNXRuntime on browsers.
+- [onnxruntime-react-native](): TBD
```
Comment (hanbitmyths) [inline js/README.md:9]: https://github.com/microsoft/onnxruntime/tree/master/js/react_native

File: js/api-usage_inference-session/package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "api-usage_inference-session",
+  "private": true,
+  "version": "1.0.0",
+  "description": "This example is a demonstration of basic usage of InferenceSession.",
+  "main": "index.js",
+  "dependencies": {
+    "onnxruntime-node": "1.7.0-dev.0"
```
Comment (hanbitmyths) [inline js/api-usage_inference-session/package.json:8]: ^1.7.0?

File: js/api-usage_tensor/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "api-usage_create-tensor",
+  "private": true,
+  "version": "1.0.0",
+  "description": "This example is a demonstration of basic usage of Tensor.",
+  "dependencies": {
+    "onnxruntime-node": "1.7.0-dev.0"
```
Comment (hanbitmyths) [inline js/api-usage_tensor/package.json:7]: same here

**Review by hanbitmyths — COMMENTED** [review]
update https://github.com/microsoft/onnxruntime-inference-examples/blob/main/README.md, too

File: js/api-usage_inference-session/package.json (hunk 2 above)
Comment (fs-eire) [reply js/api-usage_inference-session/package.json:8]: I updated them to `1.8.0`, but they will not work until 1.8.0 release.

**Comment by fs-eire** [comment]
comments resolved.

**Review by hanbitmyths — APPROVED** [review]

**Pull request merged**